src/index.html

```diff
@@ -1,18 +1,77 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Sen:wght@400..800&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Apple Monoblock"
+        class="card__photo"
+      />
+
+      <a
```

Comment:
do we need tag `a` for it? why?

Reply:
@VolodymyrKirichenko  I wanted to make this clickable. But we`ve already discussed it in today`s lesson.
Now I know that it`s much better to use a `h3` tag to highlight the title.